src/mono/netcore/System.Private.CoreLib/src/LinkerDescriptor/System.Private.CoreLib.xml

```diff
@@ -14,7 +14,8 @@
 		</type>
 
 		<!-- assembly-load-context.c: -->
-		<type fullname="System.Runtime.Loader.AssemblyLoadContext">
+		<-- object-internals.h: MonoManagedAssemblyLoadContext -->
```

Comment:
This XML comment is wrong (doesn't start with `<!--`) which likely is the reason for the test failures.

It's a bit worrying that this doesn't cause a build error.